phpstan.dist.neon

```diff
@@ -17,4 +17,5 @@ parameters:
   - src
 
   ignoreErrors:
-  - "#^Unsafe usage of new static\\(\\)\\.$#"
+   - "#^Unsafe usage of new static\\(\\)\\.$#"
+   - "#^Call to method label\\(\\) on an unknown class Spatie\\\\(LaravelRay|RayBundle|WordPressRay|YiiRay)\\\\Ray\\.$#"
```

Comment:
Looks like there was an additional space for each line added.
YAML can be sensitive at times. 😛

```suggestion
  - "#^Unsafe usage of new static\\(\\)\\.$#"
  - "#^Call to method label\\(\\) on an unknown class Spatie\\\\(LaravelRay|RayBundle|WordPressRay|YiiRay)\\\\Ray\\.$#"
```